src/coreComponents/constitutive/fluid/MultiFluidPVTPackageWrapper.hpp

```diff
@@ -21,17 +21,11 @@
 
 #include "constitutive/fluid/MultiFluidBase.hpp"
 
-// There is something wrong in the way we are building.
-// If I include, I have to add the PVT dependency so far away...
-// Therefore I forward declare...
-namespace pvt
-{
-class MultiphaseSystem;
-
-enum class PHASE_TYPE : int;
-}
-
+#include "constitutive/fluid/MultiFluidUtils.hpp"
 #include <memory>
+// For a reason that I don't understand, adding this include
+// requires adding PVTPackage as a dependent of the physicsSolvers
+#include "pvt/pvt.hpp"
```

Comment:
Moving the two `compute` functions to the hpp file requires including `pvt/pvt.hpp` in `MultiFluidPVTPackageWrapper.hpp`. If I don't add `PVTPackage` as a dependency of `physicsSolvers`, then I get the error: 
```
MultiFluidPVTPackageWrapper.hpp:25:10: fatal error: 'pvt/pvt.hpp' file not found
```
Apparently someone had run into the same issue before (see the deleted comment on GitHub). I spent some time trying to fix this but could not find a good solution. If someone knows how to make this works without making `PVTPackage` a dependency of the `physicsSolvers`, let me know.

Reply:
@bmhan12 this isn't going to make your life any easier. Sorry.

Reply:
Can't `PVTPackage` be a dependency of `constitutive`?

Reply:
`PVTPackage` was already a dependency of `constitutive`:
https://github.com/GEOSX/GEOSX/blob/848d9e5e180d0bb90309704c4f8b300daa2c10be/src/coreComponents/constitutive/CMakeLists.txt#L119-L133
and this was enough when `pvt/pvt.hpp` was included in `MultiFluidPVTPackageWrapper.cpp`. Now that `pvt/pvt.hpp` has to be included in  `MultiFluidPVTPackageWrapper.hpp`, it is not enough anymore, and the included file `pvt/pvt.hpp` cannot be found (unless `PVTPackage` is a also dependency of `physicsSolvers`).

Reply:
> Can't `PVTPackage` be a dependency of `constitutive`?

It already is, but `constitutive` isn't a dependency of `physicsSolvers`:
https://github.com/GEOSX/GEOSX/blob/848d9e5e180d0bb90309704c4f8b300daa2c10be/src/coreComponents/physicsSolvers/CMakeLists.txt#L111-L115

Reply:
In the last commit I made `physicsSolvers` depend on `constitutive` (and as before, `constitutive` itselft depends on `PVTPackage`).